projects/PorousMaterials.yaml

```diff
@@ -0,0 +1,17 @@
+Name: PorousMaterials
+Description: Calculating adsorption properties for porous materials using
+  Monte Carlo algorithms or simulating Henry constants using Widom
+  insertions. This is accomplished using a Julia code our group has developed,
+  PorousMaterials.jl
+Department: Chemical, Biological and Environmental Engineering
+FieldOfScience: Chemical Engineering
+Organization: Oregon State University
+PIName: Cory Simon
```

Comment:
We don't have any information on Cory Simon; could you email me (for privacy reasons) his email address and some brief profile information about who he is?

Reply:
Never mind, we don't keep track of PI information.